CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
   * VoiceOver improvements
   * Fix color contrast ratios
   * Increase hit area for several buttons
+* Localization
```

Comment:
Should these be under a Breaking Changes section?

Reply:
Good call, yes these are definitely breaking changes